tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/debugger.module.ts

```diff
@@ -0,0 +1,25 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {NgModule} from '@angular/core';
+
+import {DebuggerComponent} from './debugger.component';
+import {DebuggerContainer} from './debugger.container';
+
+@NgModule({
+  declarations: [DebuggerComponent, DebuggerContainer],
+  exports: [DebuggerComponent, DebuggerContainer],
```

Comment:
You do not want to export `DebuggerComponent` unless you want it to be usable by others.

Reply:
Done.